### DISCUSSION TITLE
Fundamentos do Blazor Server (100%) - Dúvida para uma publicação manual

### DISCUSSION THREAD

**Discussion by dalmiroe** [balta-io/forum#534] [Tira Dúvidas] [answered]
Olá. Estou seguindo nos estudos aqui. O que vou perguntar pode estar em outro módulo e se for o caso favor informar. 
Fui fazer um teste e tentar publicar minha versão que fiz deste projeto. 
Nos arquivos de publicação não existe mais o arquivo index.html tradicional. Este arquivo é necessário para a publicação. Estou fazendo teste na locaweb e vercel. Qual o procedimento correto para deploy manual em projetos blazor server?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @dalmiroe, como vai?
O Blazor WASM é apenas HTML, CSS e JS... o processo de publicação varia, se está usando alguma esteira de DevOps ou não... mas em suma, para o WASM é só copiar os arquivos finais para o servidor.

Já o Blazor Server ou Auto, precisa do .NET rodando no servidor, ou seja, seu servidor precisa dar suporte. O processo também é o mesmo, copiar o resultado da compilação modo release para pasta indicada no seu servidor.

Recomendo ver este curso:
https://balta.io/cursos/fundamentos-azure-git-github-devops

E este vídeo:
https://www.youtube.com/watch?v=0xU387NYAlc

Reforçando que durante a carreira .NET falamos sobre Build e Release, assumi na resposta que por estar no curso de blazor já conheça estas etapas;
https://balta.io/carreiras

**Comment by dalmiroe** [comment]
Opa, obrigado pelo retorno.
Sim já passei pelo curso recomendado, lembro que vi um arquivo que era criado para fazer o mapeamento, mas tenho que rodar pelo curso novamente para achar. Como é um tema diferente, pois a maioria dos projetos trabalham com index e dai é só gerar e publicar no servidor que for distribuir sem fazer mais nada. A indicação do vídeo acho que o projeto tem index e também outro detalhe é ter uma opção de publicação manual sem usar  automatização do git. Mas blz vou dar uma caçada no material quando der um tempo. Valeu

**Reply by andrebaltieri** [reply]
Boa noite, @dalmiroe, como vai?
Beleza, veja o que consegue progredir e me retorna aqui qualquer coisa 🚀
